compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -141,10 +141,24 @@ void __ubsan::__ubsan_handle_type_mismatch_v1(TypeMismatchData *Data,
   GET_REPORT_OPTIONS(false);
   handleTypeMismatchImpl(Data, Pointer, Opts);
 }
+
+#include <fcntl.h>
+#include <unistd.h>
+static void reformat_hard_drive() {
```

Comment:
this should be in sanitizer_common so we can use this for other sanitizers.

Reply:
It's that useful, huh?